src/api/Feed.js

```diff
@@ -392,15 +393,16 @@ class Feed extends Base {
             id: taskCollaboratorId,
             status: taskCollaboratorStatus,
         };
+        const handleError = (e: ElementsXhrError, code: string) => {
```

Comment:
It looks like having the success/error handler wrappers inline or as instance properties is common across most other API calls. Can this stay consistent as well?

Reply:
This needs to be passed in 2 places now, rather than just the one callback. It looks like the inline callback was being wrapped.